search/grammar_v2_engine.go

```diff
@@ -399,3 +487,61 @@ func (e *ESEngine) collectionHitFromSql(query string) (*elastic.SearchHit, error
 	}
 	return hit, nil
 }
+
+func retrieveTextVarValues(str string) []string {
```

Comment:
Document this function, what it does... provide example and/or general rule.

Reply:
comments added